### ISSUE TITLE
One click copy value to clipboard

### ISSUE THREAD

**Issue by keifgwinn** [3drepo/3drepo.io#1979]
We have a number of data fields available in the UI that could be enhanced with a copy to clipboard button for simple mouse over and click, rather than triple-click and keyboard shortcuts. This would make usage easier for people who need to access the GUIDs or metadata in models for use elsewhere.

Here is an example of the implementation in Azure portal.
[image](https://user-images.githubusercontent.com/23369849/79563236-4ba02600-80a4-11ea-8278-ed90cf59c977.png)

### Goals

- [ ] Add one click copy to clipboard

### Tasks

- [ ] Identify commonly copied text fields
- [ ] Add one click copy to clipboard

**Comment by MajaZubko** [comment]
@carmenfan We think that adding the copy to clipboard button in Measurements (copy name + measurement) and in BIM (copy text + value) would be a good choice. 
What do you think about it? Do you have any other text fields in mind for this enhancement?

**Comment by carmenfan** [comment]
@3drepo/product-team  any thoughts on this?

I think the main one that I use a lot is copying the model ID from model settings, that could be useful to have a copy button.

**Comment by m1chalkubiak** [comment]
@carmenfan just to be clear, @MajaZubko  prepared base component which adds copy functionality to text field. Should we add it only to model ID from model settings for now? Or should we wait for any suggestions regarding additional fields from product-team ?

**Comment by carmenfan** [comment]
> @carmenfan just to be clear, @MajaZubko prepared base component which adds copy functionality to text field. Should we add it only to model ID from model settings for now? 

I think modelID and apiKey (on user profile) would be great. @3drepo/product-team  any further thoughts?

**Comment by carmenfan** [comment]
@m1chalkubiak  what happened to the changes here? looks like @MajaZubko  made a commit, is this done?

**Comment by m1chalkubiak** [comment]
@carmenfan I referred to this topic some time ago in e-mail correspondence. The thing was that with Presentation mode, we added the same functionality to the input field. However, Presentation mode PR was not merged for a long time. Currently, we have the same functionality but it looks different, so it would be best to re-implement it according to the new look. I had it somewhere in the back of my head, but we had other priorities.

**Comment by carmenfan** [comment]
@m1chalkubiak  ah ok, fair enough! 👍

**Comment by carmenfan** [comment]
@mosment3DRepo  fyi

**Comment by m1chalkubiak** [comment]
Hi @carmenfan I have prepared a new version of this functionality, I made it locally, starting from the latest branch staging. Since I do not need the previous commits, I am wondering how to best handle this situation for this project.
Revert all previous commits, then merge staging to it, remove all previous commits and merge staging, or just remove a current branch from the repo and then push the new one?
